### DISCUSSION TITLE
Transparent background does not seem to work on 2.2.

### DISCUSSION THREAD

**Discussion by aykaramba** [lem-project/lem#1313] [General]
I downloaded 2.2.0 and compiled it from here:  https://github.com/lem-project/lem/releases/tag/v2.2.0

The following page says that lem-sdl2 supports transparent backgrounds here: https://lem-project.github.io/usage/usage/

I added this code to my init.lisp file but I cannot get it to render a transparent background: 

    (sdl2-ffi.functions:sdl-set-window-opacity (lem-sdl2/display::display-window lem-sdl2/display::*display*) 0.3)

Evaluating it in the start-lisp-repl doesn't seem to change anything in lem-sdl2 either.  

Does this work for anyone else?  I am on Debian 12.2.

**Comment by vindarel** [comment]
hello, yes it's still working for me (Mint, older Debian base).

    (sdl2-ffi.functions:sdl-set-window-opacity (lem-sdl2/display::display-window lem-sdl2/display::*display*) 0.9)

**Reply by aykaramba** [reply]
Thanks, that will help me debug.

**Comment by aykaramba** [comment]
Thank you so much for the follow up. I did a test and the setting works great on Xfce so the issue is a conflict between Stumpwm and Lem running on SBCL.

This isn't an issue and I can do without the capability.

I would like to learn about why this interaction might be. I have great interest in running Stumpwm as my window manager and then run as many common lisp applications compiled on SBCL on the same environment. What do I need to know in order to be able to run the applications in such a way that they don't interfere with each other? I have run the Kons-9 3d app on Stumpwm without issues and in the other topic starting the Sketch package via the slime repl (thank you for the info!) resolved that rendering problem. What do I need to learn about to understand the interaction between two SBCL apps here and how to run them?

**Reply by g-gundam** [reply]
You might not be running a compositor.  Try installing **[picom](https://github.com/yshui/picom)**, and then run:

```bash
picom -b
```

Then try to enable transparency in Lem.

Picom is the compositor that people who run i3wm often use.  It's probably suitable for StumpWM too.

**Reply by aykaramba** [reply]
Just dropped in to say thanks for the tip.  Picom is compatible with StumpWM and works great.  Thanks.
